test.txt

```diff
@@ -0,0 +1,16 @@
+hello
+this
+is
+a
+
+
```

Comment:
This is badly formatted! please change this

Reply:
fixed